ui/models/job.js

```diff
@@ -96,11 +97,83 @@ export default class JobModel {
     return JobModel.getList(repoName, options, config);
   }
 
-  static retrigger(repoName, jobIds) {
-    return create(`${uri}retrigger/`, { job_id_list: jobIds });
```

Comment:
After this merges I'll file a bug to remove these APIs and the other backend parts (landing these at a later date rather than with this is preferred, since will prevent errors from clients who haven't refreshed after this deploys. Also the cleanup will fit well with the rest of the buildbot cleanup I'm doing, so makes sense for me to take it)